services/varnish/run

```diff
@@ -1,8 +1,3 @@
 #!/bin/sh
-# Listen on port 80, administration on localhost:6082, and forward to
-# one content server selected by the vcl file, based on the request.
-exec docker-entrypoint.sh varnishd \
-  -F \
-  -a :80 \
-  -T localhost:6082 \
-  -f /etc/varnish/default.vcl
```

Comment:
These are all defaults now?

Reply:
> These are all defaults now?
@deviantintegral [the official entrypoint](https://github.com/varnish/docker-varnish/blob/d1212e4b8fd35b58c19b01ed389f8841d0a4ea38/stable/debian/scripts/docker-varnish-entrypoint) does this:

```
exec varnishd \
	    -F \
	    -f /etc/varnish/default.vcl \
	    -a http=:80,HTTP \
	    -a proxy=:8443,PROXY \
	    -p feature=+http2 \
	    -s malloc,$VARNISH_SIZE
```

Where VARNISH_SIZE defaults to 100M.

I was thinking that will be okay to reference directly (and more future-proof, if future major versions of varnish introduce changes). What do you think?

I believe we don't need the admin bound to `localhost:6082` because we're using varnishadm for the healthcheck instead of `nc`?

Reply:
Yes, I think that's fine!